src/nvim/highlight_group.c

```diff
@@ -248,16 +249,18 @@ static const char *highlight_init_both[] = {
   "default link @type         Type",
   "default link @type.builtin Special",
 
-  "default link @attribute Macro",
-  "default link @property  Identifier",
+  "default link @attribute         Macro",
+  "default link @attribute.builtin Special",
+  "default link @property          Identifier",
 
   "default link @function         Function",
   "default link @function.builtin Special",
 
   "default link @constructor Special",
   "default link @operator    Operator",
 
-  "default link @keyword Keyword",
+  "default link @keyword      Keyword",
+  "default link @keyword.type Structure",
```

Comment:
Any particular reason to explicitly link `@keyword.type`? At least in default color scheme that would lead for them to not be bold and be plain `Nvim*Grey2`.

If there is no strong reason to link it, I'd leave `@keyword.type` fall back to `@keyword` as all other keywords.

Reply:
The links were meant mostly as illustration (and best-effort extrapolation). I defer to you for actual links; just make a suggestion and I'll commit it.

Reply:
I'd remove explicit `@keyword.type` link and let it fall back to `@keyword`.

As other new groups have `builtin`, they should link to `Special` (as they already do, so nothing to do there).